queries.sql

```diff
@@ -1,3 +1,57 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT
+    id,
+    name,
+    date_of_birth,
+    escape_attempts,
+    neutered,
+    weight_kg
```

Comment:
- [ ] [OPTIONAL] Even though this query would work, you can use this one instead to keep your code a bit cleaner and faster:
`SELECT * FROM animals WHERE name LIKE  '%mon'`
Take into count that, when you select `*`, you are selecting everything inside the table you are, so be careful, but if you need to see all data, this is better.

Reply:
## Hi @Lordkaito,
Thanks for your feedback :+1:  but so far as I know, using the '*' selector is a bad practice, as they also say in this article: https://www.java67.com/2018/02/why-you-should-not-use-select-in -sql.html